package.json

```diff
@@ -44,25 +44,27 @@
   ],
   "dependencies": {
     "lodash": "^4.17.4",
-    "quill": "^1.2.6"
+    "prop-types": "^15.5.10",
+    "quill": "^1.3.0",
```

Comment:
Is this a hard requirement? Technically I don't see any 1.3.0-specific APIs being used.